lib/dest/prepare.js

```diff
@@ -22,6 +24,8 @@ function prepareWrite(folderResolver, optResolver) {
     var writePath = path.resolve(basePath, file.relative);
 
     // Wire up new properties
+    // TODO symlinks: new stats will have invalid props, but target stats will
```

Comment:
Don't we read the `fs.stat` from the newly created file and update or something like that?

Reply:
No, unlike the other write-xxx modules the symbolic ones (nor `symlink`) currently doesn't go through `fo.updateMetadata()`. Perhaps it should, partially, only to reflect the stats in disk.

Reply:
Yeah, I'm wondering if that helps a bunch of the problems we ran into with this implementation.

Reply:
I'll try that.